src/main/java/com/pro/sky/course2/coursework2/service/ExaminerServiceImpl.java

```diff
@@ -0,0 +1,31 @@
+package com.pro.sky.course2.coursework2.service;
+
+import com.pro.sky.course2.coursework2.domain.Question;
+import com.pro.sky.course2.coursework2.exception.AmountIsTooBigException;
+import org.springframework.stereotype.Service;
+
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Set;
+
+@Service
+public class ExaminerServiceImpl implements ExaminerService {
+    JavaQuestionService javaQuestionService;
```

Comment:
```suggestion
    private final QuestionService javaQuestionService;
```
лучше использовать интерфейсные ссылки